api/coreservice.go

```diff
@@ -156,6 +156,7 @@ type (
 		gs                *gasstation.GasStation
 		broadcastHandler  BroadcastOutbound
 		cfg               config.API
+		scheme            string
```

Comment:
this is weird that we need such a `scheme` in API. we need a better way to read state of rewarding.